awx/main/isolated/manager.py

```diff
@@ -40,6 +42,51 @@ def __init__(self, cancelled_callback=None, check_callback=None):
         self.idle_timeout = max(60, 2 * settings.AWX_ISOLATED_CONNECTION_TIMEOUT)
         self.started_at = None
         self.captured_command_artifact = False
+        self.instance = None
+
+    def build_inventory(self, hosts):
+        if self.instance and self.instance.is_containerized:
+            inventory = { 'all': { 'hosts': {} } }
+            for host in hosts:
+                inventory['all']['hosts'][host] = self.kubectl_or_oc_vars
+        else:
+            inventory = '\n'.join([
+                '{} ansible_ssh_user={}'.format(host, settings.AWX_ISOLATED_USERNAME)
+                for host in hosts
+            ])
+
+        return inventory
+
+    @property
+    def kubectl_or_oc_vars(self):
+        extravars = {}
+
+        credential = self.instance.instance_group.credential
```

Comment:
For the life of me I can't figure out why a singular `instance_group` would hang off of an `instance`. I would expect the plural version. Can you help me out?

Reply:
When I run `py.test awx/main/tests/functional/task_management/test_container_groups.py` I get `django.db.utils.OperationalError: no such column: main_instancegroup.credential_id`. Did you maybe miss adding a migration to this PR?